src/app/caesar-portal/common/models/group.ts

```diff
@@ -1,7 +1,27 @@
+import { GroupStatus } from '../models/group-status'
+
 export class Group {
-  groupId: number;
-  name: string;
-  startDate: string;
-  finishDate: string;
-  experts: string[];
+  constructor(public groupId: number, public name: string,
+    public startDate: string,
+    public finishDate: string,
+    public experts: string[]) { }
+
+  get status(): GroupStatus {
+    let currentDate = new Date(),
+      startDate = new Date(this.startDate),
+      finishDate = new Date(this.finishDate),
+      result;
+
+    if (currentDate < startDate) {
+      result = GroupStatus['Finished'];
+    }
+    else if (startDate < currentDate && currentDate < finishDate) {
+      result = GroupStatus['Current'];
```

Comment:
GroupStatus.Current, use [] to get the value only in case this value is inside a variable